src/sections/Home/Partners-home/partners-home-data.js

```diff
@@ -9,6 +9,7 @@ import rackspace from "../../../assets/images/partners/rackspace_black.svg";
 import intel from "../../../assets/images/partners/intel-grey.svg";
 import vmware from "../../../assets/images/partners/vmware.svg";
 import cncf from "../../../assets/images/partners/cncf/horizontal/black/cncf-black.svg";
+import metabittrading from  "../../../assets/images/partners/metabit-trading-logo-gray .svg";
```

Comment:
Update the path here after fixing the file name

Reply:
whitespace removed from filename.